content/source-git/status.md

```diff
@@ -0,0 +1,85 @@
+  * Being part of RHEL on GitLab initiative (being discussed).
+  * Providing tooling for CentOS Stream. (Currently being done in collaboration with Gris Ge, but the team can put more effort here.)
+  * Enhancing the current Packit workflows. Supporting internal/external teams in upstream projects (saving time by helping the shift left movement).-> Improvements in the current integration with Copr, internal/public Testing Farm, maintenance automation, VM image builds. (We can save more time/many, the numbers are being collected.)
+* What can people expect from now on?
+  * CLI supporting the basic source-git manipulation is provided as part of the `packit` Fedora/Epel8 package that can be installed in a local environment.
+    * Initializing a source-git repository, syncing the content between source-git and dist-git and obtaining the status of the sync are the supported usecases.
+  * Integration for gitlab.com/redhat/centos-stream/src is being deployed.
+    * This integration takes care of opening a new dist-git merge request for any Source git merge request and synchronising the CI statuses back.
+  * Critical fixes are being resolved to maintain the current feature set.
+  * The team will not implement any new functionality (but we encourage/help others to do it).
+  * The team will check the demand and the priority of this feature on a quarterly basis to update the priority of this effort.
+
+## FAQ
+
+### I really want this feature to be implemented. What can we do?
+You can definitely reach the team and we can discuss what needs to be done. We can either help you implement this or (in case of higher demand) Packit team will implement this for you. The Packit team will not pick any new work without strong justification.
+
+### Is Source Git dead?
+No. This is just about deprioritizing this effort. Packit team regularly checks the priorities and if the demand for this increases, the team will prioritise this again.
+
+### Can we do something to move this effort forward? 
+Yes, either bring multiple other teams and reach us, or we are willing to pass this to a new team to continue the work.
+
+### Can we do something to help this succeed in the future?
+Yes, simplify the distribution workflow itself and reduce differences between Fedora and RHEL/Stream. Ask the Red Hat “Operating System CI” (OSCI) team and/or [Veronika Kabatova](https://github.com/veruu) about how you can help.
+
+### Why don’t we remove dist git completely? Wouldn’t it be easier?
+It would be really nice, but it’s a bold move that needs to be planned and done properly to be accepted. We, as a Packit team, don’t have enough people and power to do this. This shouldn’t be a decision of the Packit team, but we can be asked to do the implementation once it’s agreed on. (Because of this, we went with an opt-in approach instead. Note the ratio of packages potentially interested in Source Git.)
```

Comment:
```suggestion
It would be really nice, but it’s a bold move that needs to be planned and done properly to be accepted. We, as a Packit team, don’t have enough people and power to do this. This shouldn’t be a decision of the Packit team, but we can be asked to do the implementation once it’s agreed on. (Because of this, we went with an opt-in approach instead. Note the ratio of packages potentially interested in source-git.)
```